spec/models/github_user_spec.rb

```diff
@@ -33,15 +33,15 @@
 
   describe "#github_avatar_url", :vcr do
     it "returns the correct url with a default size of 40" do
-      expected_url = "https://avatars2.githubusercontent.com/u/#{github_user.id}?v=4&size=40"
-      expect(github_user.github_avatar_url).to eql(expected_url)
+      expected_url = %r{https:\/\/avatars[0-9].githubusercontent.com\/u\/#{github_user.id}\?v=4&size=40\z}
```

Comment:
Added regex check since there can be multiple numbers in the avatar url (and the test started failing because of that)